src/style.css

```diff
@@ -0,0 +1,127 @@
+/* styling of the to do list project */
+
+/* body styles */
+
+body {
+  font-family: 'Roboto', sans-serif;
+  background-color: wheat;
+}
+
+h1 {
+  text-align: center;
+}
+
+/* todo tasks form and buttons */
+
+#reset:hover {
+  cursor: pointer;
+}
```

Comment:
- [ ] 🙏Please adjust the style of your app to resemble the appearance🌹 shown in [this video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Doing so will enhance the overall visual appeal of your application.

Reply:
I have implemented the changes as per your kind suggestions sir and I am really thankful to you for that.